Homeworks/HW_2/hw_2_2.py

```diff
@@ -0,0 +1,38 @@
+# Напишите программу, которая принимает две строки вида “a/b” — дробь с числителем и знаменателем.
+# Программа должна возвращать сумму и *произведение дробей. Для проверки своего
+# кода используйте модуль fractions.
+
+from fractions import Fraction
+from math import gcd
+
+# сокращение дроби
+def fraction_reduction(fractions_num:int, fractions_den:int) -> str:
+    # находим НОД
+    nod = gcd(fractions_num, fractions_den)
+    # если есть НОД > 1, то сокращаем дробь
+    if nod > 1:
+        fractions_num = int(fractions_num / nod)
+        fractions_den = int(fractions_den / nod)
```

Comment:
А чем обычный оператор целочисленного деления вас не устроил?
```suggestion
        fractions_num = fractions_num // nod
        fractions_den = fractions_den // nod
```